plugins/modules/cloud/misc/proxmox_snap.py

```diff
@@ -28,7 +29,7 @@
   state:
     description:
      - Indicate desired state of the instance snapshot.
```

Comment:
You also want to add some documentation on how that value works.
```suggestion
     - Indicate desired state of the instance snapshot.
     - The C(rollback) value was added in community.general 4.7.0.
```
You should also mention in the documentation of `snapname` that it is required if `I(state=rollback)`.